tests/model_registry/model_catalog/huggingface/test_huggingface_model_validation.py

```
@@ -52,3 +54,97 @@ def test_huggingface_model_metadata(
+    includedModels:
+    - 'ibm-granite/granite-4.0-micro*'
+""",
+                {"org_name": "ibm-granite", "included_str": "ibm-granite/granite-4.0-micro"},
+                id="test_hf_source_allowed_org_include",
+            ),
+        ],
+        indirect=True,
+    )
+    def test_hugging_face_models(
+        self: Self,
+        updated_catalog_config_map_scope_function: ConfigMap,
```

Ruff (0.14.10)

Unused method argument: `updated_catalog_config_map_scope_function`

(ARG002)